python/hsfs/util.py

```diff
@@ -233,33 +233,36 @@ def get_hostname_replaced_url(sub_path: str):
     return url_parsed.geturl()
 
 
-def verify_attribute_key_names(feature_group_obj):
+def verify_attribute_key_names(feature_group_obj, external_feature_group=False):
```

Comment:
This is unnecessary, you can just use `isinstance()` on the feature group object to check if it's an external feature group.

Reply:
I did that :) but importing FG in util class introduces circular import error